src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -390,6 +390,12 @@ class ApplicationManagerImpl
   hmi_apis::Common_DriverDistractionState::eType driver_distraction_state()
       const;
 
+  /**
+   * @brief Retrieves lock screen dismissal state (enabled/disabled).
+   * @return Current lock screen dismissal state.
+   */
+  bool is_lock_screen_dismissal_enabled_state() const;
```

Comment:
@IGapchuk rename function. Getter should have the same name as variable.

Reply:
@LuxoftAKutsan Fixed in bd9042870da8f0a7750f1b940b25411e77769044.